Explorer-Release-Setup-Instructions.md

```diff
@@ -65,16 +65,17 @@ $ docker pull storjlabs/storagenode:arm
 - `WALLET`: ethereum address for payments
 - `EMAIL`: email address so that we can notify you when a new version has been released (optional)
 - `ADDRESS`: external IP address or the DDNS you configured and the port you opened on your router `<ip>:<port>`
+   - Note: If you are using a custom port other than 28967, then you have to change the `-p 28967:28967` to `-p <port>:28967`
 - `BANDWIDTH`: how much bandwidth you want to allocate to the Storj network
 - `STORAGE`: how much disk space you want to allocate to the Storj network
 - `<identity-dir>`: the location of your identity files. You can copy the absolute path from the output of the identity commands you ran earlier
 - `<storage-dir>`: local directory where you want files to be stored on your hard drive for the network
 
 ```bash
 $ docker run -d --restart unless-stopped -p 28967:28967 \
-    -e WALLET="" \
-    -e EMAIL="" \
-    -e ADDRESS="" \
+    -e WALLET="0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" \
+    -e EMAIL="user@email.corp" \
```

Comment:
For emails and urls the standard is to use `example.com`

Reply:
Done